core/indexer/common.go

```diff
@@ -341,9 +349,39 @@ func serializeBulkTxs(
 		if err != nil {
 			log.Warn("elastic search: serialize bulk tx, write serialized tx", "error", err.Error())
 		}
+
+		if buff.Len() > txsBulkSizeThreshold {
+			buffSlice = append(buffSlice, buff)
+			buff = bytes.Buffer{}
+		}
 	}
 
-	return buff
+	// check if in the last buffer exits data
```

Comment:
"check if the last buffer contains data" sounds better I think

Reply:
done